src/run-on-arch.sh

```diff
@@ -37,7 +38,7 @@ install_deps () {
   #            linux/386, linux/arm/v7, linux/arm/v6
   sudo apt-get update -q -y
   sudo apt-get -qq install -y qemu qemu-user-static
-  docker run --rm --privileged multiarch/qemu-user-static --reset -p yes
+  docker run --rm --privileged multiarch/qemu-user-static --reset -p yes --credential yes
```

Comment:
What does this --credential do? Can't find it in the docs

Reply:
```suggestion
  docker run --rm --privileged multiarch/qemu-user-static --reset -p yes
```
tbh I can't remember why I added this and sadly I stashed my history before creating this pr. according to the docs:

> if yes, credential and security tokens are calculated according to the binary to interpret

https://github.com/multiarch/qemu-user-static

I mean: I don't have an idea why I added that. I will just remove it and check if everything I do still works.